specification/dimensions/pricingunit.md

```diff
@@ -0,0 +1,33 @@
+# Pricing Unit
+
+The Pricing Unit is a measurement unit specified by the provider for determining unit prices associated with the usage or the purchase of a specific resource and/or service. Common examples include the number of hours a compute appliance was running for a given period, the number of gigabytes * the number of hours (gigabyte-hours) for a storage appliance, or a cumulative count of requests for a network appliance or API service. For certain services providers can choose block pricing i.e., million requests as a base unit. Pricing Unit is complementary to and describes how [Quantity In Pricing Unit](#quantityinpricingunit) is measured. The Pricing Unit is primarily focused on pricing and cost and it should not be confused with the [Usage Unit](#usageunit), which is dedicated to tracking resource and/or service consumption and is often at a lower granularity than the Pricing Unit.
```

Comment:
- Improve the PricingUnit and UsageUnit definitions, especially the PricingUnit - UsageUnit cross-references by incorporating the following suggestions: 
  - _The Pricing Unit indicates how a provider rates the measured usage and purchase quantities after pricing rules are applied. It may differ from the physical measurement unit or Usage Unit due to pricing rules, such as block pricing. The Pricing Unit is commonly used for scenarios like verifying unit prices against the price sheet and should not be directly correlated with the physical resource or service usage._
  - _The Usage Unit indicates how service usage and purchases are measured by the provider. The Usage Unit is a standard measure of physical quantity before the pricing has been applied. The Usage Unit describes and is used in conjunction with Quantity In Usage Unit to gain an understanding of how much of a given product or service was used or purchased. Usage Unit may be a lower-level granularity than Pricing Unit, which applies provider-specific pricing rules. The Usage Unit is commonly used for scenarios like verifying how much of a resource or service was used with external metrics and logging systems to optimize usage patterns and should not be used to correlate directly to the unit price._
- Ensure that the definitions of related columns (PricingUnit, QuantityInPricingUnit, UsageUnit, and QuantityInUsageUnit) are aligned.
- Discuss these changes with both VS1 and VS3 team members.

Reply:
First 2 points covered, discussion was taken and maintainers reviews should cover the "discussion" part